install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -220,7 +220,8 @@ spec:
                 done
               else
                 # COS-beta (with containerd)
-                for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do crictl stopp $(cat $f) || true; done
+                which crictl || PATH=${PATH}:/home/kubernetes/bin
```

Comment:
- FYI: `which` is not a shell built-in (at least not in Bash), the convention is to use `command -v` (see https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script for explanation).
- Checking whether or not `crictl` exists here is arguably a bit pointless. There's no harm in having `/home/kubernetes/bin` in PATH even on older COS installs where this dir is not present.
- Please add a comment stating why this PATH extension is done. (e.g. required as of COS version x.y.z or any other useful info you might have) This helps with maintenance.

Thanks!

Reply:
Thanks, I wasn't aware that `which` wasn't recommended. I will make the requested changes.